ggml/src/ggml-vulkan/vulkan-shaders/mul_mat_split_k_reduce.comp

```diff
@@ -5,25 +5,44 @@
 layout(local_size_x = 256, local_size_y = 1, local_size_z = 1) in;
 
 layout (binding = 0) readonly buffer A {float data_a[];};
+layout (binding = 0) readonly buffer A4 {vec4 data_a4[];};
 layout (binding = 1) writeonly buffer D {float data_d[];};
+layout (binding = 1) writeonly buffer D4 {vec4 data_d4[];};
```

Comment:
This double binding is very cool, I wasn't aware this was possible.

Reply:
Yes, it's legal, the spec even calls it out in a note:

```
If multiple shader variables are declared with the same set and binding
values, and with the same underlying descriptor type, they can all be
statically used within the same shader.
```